R/object_name_linters.R

```diff
@@ -239,6 +240,21 @@ is_base_function <- function(x) {
   x %in% base_funs
 }
 
+# see ?".onLoad" and ?"Startup"
+special_funs <- c(
+  ".onLoad",
+  ".onAttach",
+  ".onUnload",
+  ".onDetach",
+  ".Last.lib",
+  ".First",
+  ".Last"
+)
+
+is_special_function <- function(x) {
```

Comment:
hmm I guess we need a test for this :sweat_smile: I'm seeing this on current master when trying to do #597:

```
R/zzz.R:185:1: style: Variable and function name style should be snake_case.
.onLoad <- function(libname, pkgname) {
^~~~~~~
```

Reply:
The issue is that `strip_names` removes the leading `.`. So actually we need to search for `onLoad` etc.